packages/palette/src/elements/Radio/Radio.tsx

```diff
@@ -41,66 +37,72 @@ export interface RadioToggleProps
 
 /**
  * A Radio button
- *
- * Spec: zpl.io/bAvnwlB
  */
-export const Radio: React.SFC<RadioProps> = props => {
-  const {
-    children,
-    disabled,
-    hover,
-    name,
-    onSelect: _onSelect,
-    selected,
-    value,
-    label,
-    ...others
-  } = props
-
-  // Ensures that only one call to `onSelect` occurs, regardless of whether the
-  // user clicks the radio element or the label.
-  const onSelect = _onSelect && debounce(_onSelect, 0)
-
-  return (
-    <Container
-      disabled={disabled}
-      alignItems="center"
-      selected={selected}
-      hover={hover}
-      onClick={() =>
-        !disabled && onSelect && onSelect({ selected: !selected, value })
-      }
-      {...others}
-    >
-      <RadioButton
-        role="presentation"
-        border={1}
-        mr={1}
-        mt="2px"
-        mb="-2px"
```

Comment:
Said weird margin thing. I'm assuming these were to somehow negate the border? Not sure; they mess up alignment.